call-notes/call_5.md

```diff
@@ -0,0 +1,152 @@
+
+**Maximilian Schneider:** Yeah that makes sense. I think this limit is a bit surprising. At least for us it was a bit surprising. There is another one I think it is the receive window size that I think have caused some discussion a few times and I think the idea that was like you have basically based on the stake different we receive window sizes which limits the number of parallel streams there can be per connection. So, it's like eight connections per identity and then I think there is some depending on the stake there, each connection can have a certain number of stream.
+
+**Stephen Akridge:** Right, yeah the eight connections is just to kind of if you had clients behind like a router or you had a condition where you got disconnected and you needed to reconnect you might have some connections overlapping. So just not to like kick you out immediately if you had a stale connection in the connection pool. But yeah, we are using the streams to throttle based on stake and then of course a budget for unstaked as well. And those are subject to tweaking, I think we wanted to roll out of the first version and then see kind of monitor the metrics and then update them potentially as we see the use in the validator. I don't think they are, I mean they seem to have been fairly reasonable defaults but open to tweaking I think.
+
+**Jacob Creech:** Just a question for any of people from the Firedancer team. I know that you all have been working on QUIC. Did you all choose similar like received window parameters in tuning or have you all not gotten to that part yet?
+
+**Kelvin Bowers:** I don't think Nick is on this call, he had be the best person to answer it. Let me go see if I can find Nick.
+
+**Jacob Creech:** Okay, thank you.
+
+**Tom Pointon:** I think he's on vacation today.
+
+**Kelvin Bowers:** Oh then, I probably won't be finding Nick. I don't have the answer to that off top my head.
+
+**Jacob Creech:** Okay, maybe probably we probably should sync as we go further into the implementation of like what are the different parameters that we have seen because in the future I think as Stephen as you said it was a good default to start with but what is a better performance in the long run would be nice.
+
+**Stephen Akridge:** Saw a question in the chat real fast about the state-based variables. So you have more stake generally gets more bandwidth and more resources and you know say of transactions into the validator. So that's the idea.
```

Comment:
```suggestion
**Stephen Akridge:** Saw a question in the chat real fast about the state-based variables. So more stake generally gets more bandwidth and more resources and you know say of transactions into the validator. So that's the idea.
```